dcrutil/wif.go

```diff
@@ -46,27 +51,17 @@ type WIF struct {
 // as a string encoded in the Wallet Import Format.  The compress argument
 // specifies whether the address intended to be imported or exported was created
 // by serializing the public key compressed rather than uncompressed.
-func NewWIF(privKey chainec.PrivateKey, net *chaincfg.Params, ecType dcrec.SignatureType) (*WIF,
-	error) {
-	if net == nil {
-		return nil, errors.New("no network")
-	}
-	return &WIF{ecType, privKey, net.PrivateKeyID}, nil
-}
-
-// IsForNet returns whether or not the decoded WIF structure is associated
-// with the passed network.
-func (w *WIF) IsForNet(net *chaincfg.Params) bool {
-	return w.netID == net.PrivateKeyID
+func NewWIF(privKey chainec.PrivateKey, net [2]byte, ecType dcrec.SignatureType) (*WIF, error) {
```

Comment:
there's no reason afaik to keep this error return.

Reply:
Good point.  I didn't remove it since it was an older change, but since this is a v2 module we can break the API accordingly.  Will update.

Reply:
Done.